crypto/bn/bn_exp.c

```diff
@@ -1040,27 +1038,42 @@ int BN_mod_exp_mont_consttime(BIGNUM *rr, const BIGNUM *a, const BIGNUM *p,
             }
         }
 
-        bits--;
-        for (wvalue = 0, i = bits % window; i >= 0; i--, bits--)
-            wvalue = (wvalue << 1) + BN_is_bit_set(p, bits);
+        /* The exponent may not have a whole number of fixed-size windows.
```

Comment:
Styling nit. Preferred style for multi-line comment is to open with `/*` on line of its own. This naturally applies to another block comment further down.